### ISSUE TITLE
Is dotnet_style_parentheses_* supported?

### ISSUE THREAD

**Issue by marcospgp** [dotnet/vscode-csharp#5356]
I can't seem to get it to work with Omnisharp in VSCode.

My .editorconfig file is working, as I can disable rules normally. The lines

```
dotnet_style_parentheses_in_arithmetic_binary_operators = always_for_clarity
dotnet_style_parentheses_in_other_binary_operators = always_for_clarity
dotnet_style_parentheses_in_other_operators = never_if_unnecessary
dotnet_style_parentheses_in_relational_binary_operators = always_for_clarity
```

however, seem to not have any effect.

Is setting these values supported?

**Comment by JoeRobich** [comment]
Code style rules require `omnisharp.enableRoslynAnalyzers` be enabled. To change the severity of these analyzers you would need to also configure IDE0047 & IDE0048. [See the docs](https://docs.microsoft.com/en-us/dotnet/fundamentals/code-analysis/style-rules/ide0047-ide0048).

Example making them warnings:
```ini
root = true

[*.{cs,vb}]
dotnet_style_parentheses_in_arithmetic_binary_operators = always_for_clarity
dotnet_style_parentheses_in_other_binary_operators = always_for_clarity
dotnet_style_parentheses_in_other_operators = never_if_unnecessary
dotnet_style_parentheses_in_relational_binary_operators = always_for_clarity

dotnet_diagnostic.IDE0047.severity = warning
dotnet_diagnostic.IDE0048.severity = warning
```

**Comment by marcospgp** [comment]
Thanks for the quick response @JoeRobich !

I did double check the things you mentioned and could not find any issue, yet the `dotnet_style` options are still not working for me :(

My `.editorconfig` file [is here ](https://gist.github.com/marcospgp/cd291e7124e663635d3a636af9c55c34) for reference.

I did try both including the `:warning` suffix (as in `always_for_clarity:warning`) and not. Neither seems to work.
